src/views/NannysRoom.vue

```diff
@@ -202,7 +202,6 @@
             </v-btn>
             <v-form
               v-if="isAddingInstructor"
-              v-model="validInstructor"
```

Comment:
This line was generating a console error: `[Vue warn]: Property or method "validInstructor" is not defined on the instance but referenced during render.`